### PR TITLE
Cleanup: Remove unused variable

### PR DESCRIPTION
**Pull request by slrtbtfs**
Signed-off-by: Tobias Guggenmos <tguggenm@redhat.com>

### CODE REVIEW

File: promql/parse.go [hunk 1]
```diff
@@ -29,9 +29,8 @@ import (
 )
 
 type parser struct {
-	lex     *Lexer
-	token   Item
-	peeking bool
```
Comment (slrtbtfs) [inline promql/parse.go]: Due to the parser rewrite `peeking` is always `false` now. So it can be as well removed.

**Comment by brian-brazil** [comment]
￼Thanks!

**Pull request merged**